hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -182,6 +183,17 @@ private void applyDBDefinition(DBDefinition definition) {
     }
   }
 
+  public DBStoreBuilder setMaxNumberOfOpenFiles(Integer maxNumberOfOpenFiles) {
+    this.maxNumberOfOpenFiles = maxNumberOfOpenFiles;
+    return this;
+  }
+
+  private void setDBOptionsProps(ManagedDBOptions dbOptions) {
```

Comment:
Neither `setDBOptionsProps` function is used anywhere nor `dbOptions` passed is used.

Reply:
done

Reply:
I think `setMaxOpenFiles` would be better naming. Naming `setDBOptionsProps` is generic and it is not doing that.

Reply:
yeah if we want add more props going forward we can just add it here in this function